generators/entity-client/templates/react/src/main/webapp/app/entities/entity.tsx.ejs

```diff
@@ -129,6 +121,7 @@ export const <%= entityReactName %> = (props: I<%= entityReactName %>Props) => {
         ...paginationState,
         activePage: 1
       });
+      props.getSearchEntities(search, paginationState.activePage - 1, paginationState.itemsPerPage, `${paginationState.sort},${paginationState.order}`);
```

Comment:
Tiny thing; missing a space between `${paginationState.sort}` and `${paginationState.order}`

Reply:
there is no space but a comma between ${paginationState.sort} and ${paginationState.order} 🧐

Reply:
Yes I am thinking we need to add a space after the comma. 🙃

Reply:
I tried but we can't because this string is used in the request
![image](https://user-images.githubusercontent.com/9989211/75631605-9b11bc00-5bf4-11ea-9014-ae8084e1aa4d.png)

Reply:
Ops you are right. My mistake. 🙃

Reply:
😉